### DISCUSSION TITLE
Hopscotch - can someone pls let me know how to generate UUID(v4) in pre-request script of Hoppscotch ? The below is what we have at the moment , was wondering if there are any libs we can import and make the below shorter ?

### DISCUSSION THREAD

**Discussion by nagumanangi1234** [hoppscotch/hoppscotch#3916] [General]
var lut = [];
 
for (var i=0; i<256; i++) { 
  lut[i] = (i<16?'0':'')+(i).toString(16); 
}
 
function e7()
{
    var d0 = Math.random()*0xffffffff|0;
    var d1 = Math.random()*0xffffffff|0;
    var d2 = Math.random()*0xffffffff|0;
    var d3 = Math.random()*0xffffffff|0;
    return lut[d0&0xff]+lut[d0>>8&0xff]+lut[d0>>16&0xff]+lut[d0>>24&0xff]+'-'+
    lut[d1&0xff]+lut[d1>>8&0xff]+'-'+lut[d1>>16&0x0f|0x40]+lut[d1>>24&0xff]+'-'+
    lut[d2&0x3f|0x80]+lut[d2>>8&0xff]+'-'+lut[d2>>16&0xff]+lut[d2>>24&0xff]+
    lut[d3&0xff]+lut[d3>>8&0xff]+lut[d3>>16&0xff]+lut[d3>>24&0xff];
}
 
const guid = e7();
 
pw.env.set("guid", guid);

**Comment by shipko** [comment]
Hi, @nagumanangi1234 

In this PR I added replaceIn method that contain this functional

https://github.com/hoppscotch/hoppscotch/pull/3993
